src/cim/components/ClusterDeployment/ArmCheckbox.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { Checkbox, FormGroup, Tooltip } from '@patternfly/react-core';
+import { useField, useFormikContext } from 'formik';
+import {
+  getFieldId,
+  isArmArchitecture,
+  ClusterCreateParams,
+  HelperText,
+  PopoverIcon,
+  CpuArchitecture,
+  OpenshiftVersionOptionType,
+} from '../../../common';
+import {
+  FeatureSupportLevelBadge,
+  useFeatureSupportLevel,
+} from '../../../common/components/featureSupportLevels';
+import { useTranslation, useTranslation } from '../../../common/hooks/use-translation-wrapper';
```

Comment:
There is TS error here
```suggestion
import { useTranslation } from '../../../common/hooks/use-translation-wrapper';
```

Reply:
Fixed